### PR TITLE
Remove PINE A64 support from codebase.

### PR DESCRIPTION
**Pull request by holgerfriedrich**
* clean clodebase from exceptions for pine64 platform
* remove pine64 image creation from build.bash
* adapt documentation
* leave platform detection for displaying a deprecation warning
* remove rc.local
* remove webif cleanup in first-boot.bash

Signed-off-by: Holger Friedrich <mail@holger-friedrich.de>

### CODE REVIEW

**Comment by ecdye** [comment]
@holgerfriedrich Make sure you rewrite `build.bash local-test` to make use of `openhabian-installer.service` now that we no longer need to rely on `rc.local` for any of our supported platforms.

**Comment by holgerfriedrich** [comment]
@ecdye thanks, overlooked this in spite of the reminder you wrote in the other ticket
@mstormi do you think we need the local-test functionality? I never use it....

**Comment by holgerfriedrich** [comment]
just pushed local-test function, it was broken since the changes for offline install anyway. untested.

**Comment by ecdye** [comment]
We use the local-test function in the Travis tests and other Docker tests so yes we use it all the time.

**Comment by ecdye** [comment]
@holgerfriedrich Don't forget to check `includes/*` for any pine64 specific code / files.

**Comment by holgerfriedrich** [comment]
> @holgerfriedrich Don't forget to check `includes/*` for any pine64 specific code / files.

Got it, thanks. Now all files from include directory are in use.

File: build.bash [hunk 1]
```diff
@@ -148,25 +145,21 @@ elif [ "$1" == "rpi64" ]; then
...
   # Use local filesystem's version of openHABian
-  sed -i 's|if \[ -d /opt/openhabian/ \]; then cd /opt && rm -rf /opt/openhabian/; fi||' /boot/first-boot.bash
-  # shellcheck disable=SC2016
-  sed -i 's|if git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi||' /boot/first-boot.bash
+  sed -i 's|openhabian_update|true|' /boot/first-boot.bash
   chmod +x /boot/first-boot.bash
   chmod +x /boot/webif.bash
-  chmod +x /etc/rc.local
-  echo_process "Local system ready for installation test. Run script /etc/rc.local or reboot to initiate"
+  echo_process "Local system ready for installation test. Run systemctl start openhabian-installer or reboot to initiate"
```
Comment (ecdye) [inline build.bash:162]: I would change "Run systemctl start openhabian-installer or ..." to "Run 'systemctl start openhabian-installer.service' to make it clear what to run.

File: functions/openhab.bash [hunk 2]
```diff
@@ -72,7 +72,7 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   cond_redirect systemctl daemon-reload
   if cond_redirect systemctl enable --now openhab2; then echo "OK"; else echo "FAILED (service)"; return 1; fi
 
-  if is_pi || is_pine64; then
+  if is_pi ; then
```
Comment (ecdye) [inline functions/openhab.bash:75]: Remove extra space before `;`

File: functions/openhabian.bash [hunk 3]
```diff
@@ -157,7 +157,7 @@ system_check_default_password() {
   if is_pi && id -u pi &>/dev/null; then
     USERNAME="pi"
     PASSWORD="raspberry"
-  elif is_pi || is_pine64; then
+  elif is_pi ; then
```
Comment (ecdye) [inline functions/openhabian.bash:160]: Same here: remove extra space before `;`

File: .travis.yml [hunk 4]
```diff
@@ -53,7 +53,7 @@ jobs:
         - docker build -t openhabian/${HW}-openhabian -f Dockerfile.amd64 .
         - echo -e "\n\e[36mopenHABian test installation using 64bit Azul Enterprise Java 8 running on a\e[31m 64bit OS\e[36m running in a virtual ${HW}...\e[0m\n"
         - docker run --name "openhabian-${HW}" -d openhabian/${HW}-openhabian;
-        - docker exec -i "openhabian-${HW}" bash -c './build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local'
+        - docker exec -i "openhabian-${HW}" bash -c './build.bash local-test && mv ~/.profile ~/.bash_profile && /boot/first-boot.bash'
```
Comment (mstormi) [inline .travis.yml:56]: not really PINE related but ok

**Pull request merged**